src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -40,8 +41,10 @@
 		put(ModElementType.GUI, Arrays.asList(new GUIBindingInverter(), new GUICoordinateConverter()));
 		put(ModElementType.PROCEDURE, Arrays.asList(new ProcedureEntityDepFixer(), new OpenGUIProcedureDepFixer(),
 				new ProcedureGlobalTriggerFixer()));
-		put(ModElementType.BIOME, Arrays.asList(new BiomeSpawnListConverter(), new BiomeDefaultFeaturesConverter()));
+		put(ModElementType.BIOME, Arrays.asList(new BiomeSpawnListConverter(), new BiomeTreeConverter(),

```

Comment:
You are still putting biome twice into the map. Please check how Java Maps work and how they handle keys and insertions to the key of the same name.